Resources/Locale/en-US/reagents/meta/consumable/drink/drinks.ftl

```diff
@@ -76,3 +76,10 @@ reagent-desc-pilk = A sickening mixture of milk and cola.
 reagent-name-posca = posca
 reagent-desc-posca = Poor warriors' drink from a forgotten era.
 
+
+reagent-name-kvass = kvass
+reagent-desc-kvass = A cool refreshing drink with a taste of socialism.
```

Comment:
Just need to remove this since it was just recently removed.

Reply:
why was it removed out of curiosity?

Reply:
I'm not sure myself.